server/lib/JJ.pm

```diff
@@ -0,0 +1,31 @@
+package JJ;
+
+use strict;
+use warnings;
+
+use Data::Dumper;
+
+our $VERSION = '0.1';
+
+=head1 NAME
+
+JJ
+
+=head1 DESCRIPTION
+
+Simple debugging helper that logs in BBEdits Logs directory.
+
+=cut
+
+sub jjlog {
```

Comment:
This file must be tidied using these perltidy args:

`-csc -gnu -l200`

Reply:
Noted